resources/assets/components/WithReviewing/index.js

```diff
@@ -149,29 +151,51 @@ const reviewComponent = (Component, data) => {
       });
     }
 
-    // Update a signups quanity.
-    updateQuantity(signup, newQuantity) {
-      // Fields to send to /posts
-      const fields = {
-        northstar_id: signup.northstar_id,
-        campaign_id: signup.campaign_id,
-        campaign_run_id: signup.campaign_run_id,
-        quantity: newQuantity,
-      };
+    // Update a post's or signup's quantity.
+    updateQuantity(postOrSignup, newQuantity) {
+      // If v3QuantitySupport is true, update individual post's quantity.
+      if (window.ENV['DS_ENABLE_V3_QUANTITY_SUPPORT']) {
+        // Field to send to /api/v3/posts/:post_id
+        const field = {
+          quantity: parseInt(newQuantity),
+        };
 
-      // Make API request to Rogue to update the quantity on the backend
-      let request = this.api.post('posts', fields);
+        // Make API request to Rogue to update the quantity on the backend
+        let request = this.api.patch('api/v3/posts/'.concat(postOrSignup['id']), field);
```

Comment:
hmm I am a little confused on all of this. If we are using `V3` endpoints for this then the feature flag doesn't need to be checked. V3 endpoints should all be considering quantity on the post. 

For example, we should never be making a request to `api/v3/post/:signup_id`

Reply:
@sbsmith86 right (and sorry this is all a little jumbled!)! I didn't want to break anything before the feature flag is turned on so that's why did the check to see if it is on or not so it works when it is both on and off.

Reply:
The feature flag should be relatively temporary. Let's add some `@TODO`s to remind us that we need to remove support for both ways since the world we want to move towards is simply quantity on the post and rogue admin using v3 endpoints.

Reply:
sounds good! I'll add those in all places we check for the feature flag now!

Reply:
is this something I should add to the list of things to refactor? I think you started a doc but unsure where it is?

Reply:
@chloealee I started a list [here](https://github.com/orgs/DoSomething/teams/team-bleed/discussions/14). We certainly need refactor all of this. One of the points in that list is to switch the rogue frontend to use V3 endpoints and we will need to figure out a way to break up that work reasonably. Maybe add this as a sub-bullet to that point so we can track it as a discrete piece of work that needs to happen.

Reply:
awesome, thanks @sbsmith86 ! Updated that :)